.travis.yml

```diff
@@ -1,15 +1,29 @@
+dist: trusty
+sudo: false
 language: php
 php:
   - 5.4
   - 5.5
   - 5.6
-  - hhvm
   - 7
+  - 7.1
+  - nightly
+
 before_script:
   - composer self-update
   - composer install
 script:
-  - phpunit --coverage-clover=coverage.clover
+  - ./vendor/bin/phpunit --coverage-clover=coverage.clover
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
+
+jobs:
+  include:
+    - stage: travis
+      dist: precise
```

Comment:
Hi @jakejohns , 

I was looking at your issue on https://github.com/auraphp/Aura.Html/pull/54#issuecomment-316886798 , thought I could experiment a bit on it.

But looks like I failed for the current time. I will do some more experiments to see if there is a way to use multiple distro on a single travis file.